### DISCUSSION TITLE
Text-Hight/Size/Color in a ElevatedButton

### DISCUSSION THREAD

**Discussion by byteman3** [flet-dev/flet#1167] [Q&A]
### Question

How can I change the Text-High/Size/Color in a ElevatedButton?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
You could set the [`.content`](https://flet.dev/docs/controls/elevatedbutton#content) property of the `ElevatedButton` to a [`Text`](https://flet.dev/docs/controls/text#color). You will then have the possibility to modify everything more freeely.

**Reply by byteman3** [reply]
### Where can I change the Button-Text Size before and by on_hover(e)?

`import flet as ft
from flet import (colors, TextStyle, Column, Container, border_radius, BorderSide, RoundedRectangleBorder)

def main(page:ft.Page):
    page.title = "Test App"
    page.window_width = 200
    page.window_height = 200

    sty = ft.ButtonStyle(
        color={
        ft.MaterialState.HOVERED: ft.colors.BLUE,
        ft.MaterialState.FOCUSED: ft.colors.YELLOW,
        ft.MaterialState.DEFAULT: ft.colors.BLACK,
        },
        bgcolor={
            ft.MaterialState.HOVERED: ft.colors.BLACK38 ,
            ft.MaterialState.FOCUSED: ft.colors.BLUE,
            ft.MaterialState.DEFAULT: ft.colors.YELLOW_200,
        },
        
        overlay_color=ft.colors.TRANSPARENT,
        elevation={"pressed": 0, "": 22},
        animation_duration=200,
        side= BorderSide(1, ft.colors.BLUE_GREY),
        shape=RoundedRectangleBorder(radius=10),  
    )


    def on_hover(e):
            e.control.bgcolor = "#404050" if e.data == "true" else "#DBE09D"
            e.control.text = "yyyyyyyyy" if e.data == "true" else "1"
            e.control.update() 

    page.add(
        ft.ElevatedButton(text="window", style = sty, on_hover=on_hover),
    )

ft.app(target=main)
`
